src/ert/data/loader.py

```diff
@@ -118,11 +118,10 @@ def _load_general_response(
             for key in facade.all_data_type_keys()
             if facade.is_gen_data_key(key) and data_key in key
         ]
-        data = pd.DataFrame()
-
+        gen_datas: List[pd.DataFrame] = []
         for time_step in time_steps:
-            gen_data = facade.load_gen_data(ensemble, data_key, time_step).T
-            data = data.append(gen_data)
+            gen_datas.append(facade.load_gen_data(ensemble, data_key, time_step).T)
```

Comment:
Why dataS in plural?

Reply:
Because it is multiple "gen_data" objects, as given by the typing specification.

Reply:
sure, but the ordinary definition for data referring to `The word data is a plural noun so write "data are`, I would think that we keep `gen_data` unless data refers to something different than the usual.
I don't think this is the case here, but maybe you know a specific reason?

Reply:
I was more thinking of "gen_data" as a specific thing, for which these rules don't apply. But I can rename it to `gen_data_list` instead.

Reply:
Nice!